build.gradle

```diff
@@ -25,16 +23,17 @@ ext {
             spock    : "org.spockframework:spock-core:$spockVersion",
             cglib    : "cglib:cglib-nodep:$cglibVersion",
             objenesis: "org.objenesis:objenesis:$objenesisVersion"
-
     ]
 
-    javadocFolder = "doc"
-
     sharedManifest = manifest {
         attributes("Implementation-Title": 'Twingly Search API Java',
                 "Implementation-Version": version)
```

Comment:
What do you mean ? What block exactly ?
`javadocFolder = "doc"` is part of obsolete functionality.
And all new magic is done by versionProperties.load